src/api_proxy/service_control/request_builder.cc

```diff
@@ -1364,6 +1366,19 @@ Status RequestBuilder::ConvertCheckResponse(
         check_response.check_info().consumer_info().project_number());
   }
 
+  if (check_response.check_info().consumer_info().consumer_number() > 0) {
+    check_response_info->consumer_number = std::to_string(
+        check_response.check_info().consumer_info().consumer_number());
+  }
+
+  if (check_response.check_info().consumer_info().type() !=
+      CheckResponse_ConsumerInfo_ConsumerType::
+          CheckResponse_ConsumerInfo_ConsumerType_CONSUMER_TYPE_UNSPECIFIED) {
```

Comment:
Would prefer `CheckResponse::ConsumerInfo::ConsumerTypeCase::CONSUMER_TYPE_UNSPECIFIED`. Then you don't need the extra using statement either.

Reply:
That is so long.
Also consider both the enum and the util function converting enum to string need that namespace, so it is fine to add `using`.